src/core/reactor.cc

```diff
@@ -167,6 +167,7 @@ module seastar;
 #include <seastar/util/defer.hh>
 #include <seastar/util/log.hh>
 #include <seastar/util/memory_diagnostics.hh>
+#include <seastar/util/noncopyable_function.hh>
```

Comment:
why? presumably `reactor.cc` always include `reactor.hh`.

Reply:
`reactor.cc` also uses this header regardless of `reactor.hh` in `network_stack_factory` impl.